### PR TITLE
chore: update versions of components

### PR DESCRIPTION
**Pull request by smira**
Switch to ARG to keep the environment clean.

Install `buildx` from the official release.

Bump versions.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
 
-FROM alpine:3.9
+FROM alpine:3.11
 
-ENV BUILDX_VERSION=v0.3.0
-ENV GITMETA_VERSION=v0.1.0-alpha.3
-ENV CLOUD_SDK_VERSION=258.0.0
-ENV CNI_PLUGINS_VERSION=v0.8.5
-ENV FIRECRACKER_VERSION=v0.21.0
+ARG GITMETA_VERSION=v0.1.0-alpha.3
```
Comment (andrewrynhard) [inline Dockerfile:27]: I don't think we depend on gitmeta at all anymore.

Comment (smira) [reply Dockerfile:27]: do we use `docker-compose` anywhere?

Comment (andrewrynhard) [reply Dockerfile:27]: I don't think we do.

Comment (smira) [reply Dockerfile]: removed those two to make it a bit more sllim

**Comment by smira** [comment]
https://ci.dev.talos-systems.io/talos-systems/build-container/67 this one is green

**Pull request merged**